templates/hardware.html

```diff
@@ -0,0 +1,153 @@
+{% extends '_layout.html' %} {% block title%} Ubuntu Desktop certified hardware
+{% endblock %} {% block content %}
+
+{{ details.release_details }}
```

Comment:
you seem to have left some debugging in here:

![image](https://user-images.githubusercontent.com/519935/62471251-385a5e00-b794-11e9-8a79-bee59f412b2f.png)

Reply:
uhh